ferrocene/doc/user-manual/src/rustfmt/rustfmt-cli.rst

```diff
@@ -0,0 +1,86 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Command-Line Interface
+======================
+
+.. cli:program:: rustfmt
+
+   .. cli:option:: --check
+
+      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
+
+   .. cli:option:: --emit [files|stdout|coverage|checkstyle|json]
```

Comment:
This needs to document what each of the possible value does, and when you might want to use each possible value.

Reply:
Okay. Right now I just copied the contents from `rustfmt --help`.